source/basic.tex

```diff
@@ -3669,10 +3669,17 @@
 
 \pnum
 \indextext{storage duration!static}%
-All variables which do not have dynamic storage duration, do not have thread
-storage duration, and are not local
-have \defn{static storage duration}. The
-storage for these entities lasts for the duration of the
+All variables which
+\begin{itemize}
+\item
+do not have thread storage duration and
+\item
+belong to a namespace scope\iref{basic.scope.namespace} or
+are declared with
+the \keyword{static} or \keyword{extern} keywords\iref{dcl.stc}
+\end{itemize}
```

Comment:
We need to exclude `static thread_local` here.

Reply:
@zygoloid, I did. The bullet just above says "does not have thread storage duration and". "static thread_local" has thread storage duration per [basic.stc.thread], so that's covered.

Reply:
Ah, right, OK.